test/e2e/multihoming.go

```diff
@@ -637,11 +691,14 @@ func netCIDR(netCIDR string, netPrefixLengthPerNode int) string {
 	return fmt.Sprintf("%s/%d", netCIDR, netPrefixLengthPerNode)
 }
 
-func popCIDRsPerNodePrefix(netCIDR string, netPrefixLengthPerNode int) string {
-	if strings.Count(netCIDR, "/") > 1 {
-		return strings.ReplaceAll(netCIDR, fmt.Sprintf("/%d", netPrefixLengthPerNode), "")
+func getNetCIDRSubnet(netCIDR string) (string, error) {
+	subStrings := strings.Split(netCIDR, "/")
+	if len(subStrings) == 3 {
```

Comment:
can you provide an example of this ? Maybe I'm missing something

Reply:
for example for ipv6, it is either 10.128.0.0/16/24 for layer3 networks, or 10.128.0.0/24 for layer2 networks.

Reply:
Right, my bad.